pkg/controllers/resources/ingresses/syncer.go

```diff
@@ -29,10 +42,18 @@ func NewSyncer(ctx *synccontext.RegisterContext) (syncertypes.Object, error) {
 		return nil, err
 	}
 
+	physicalClusterClient, err := kubernetes.NewForConfig(ctx.PhysicalManager.GetConfig())
```

Comment:
replace it with `physicalClusterClient: ctx.PhysicalManager.GetClient()`

Reply:
Done!